docs/05-technical-reference/00-architecture/ra-01-runtime-agent-workflow.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: Runtime Agent Architecture
 type: Architecture
-
 ---
 
+>**CAUTION:** Runtime Agent is used in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md) of Application Connectivity. To learn how to install Compass and integrate with it, refer to the [Compass documentation](https://github.com/kyma-incubator/compass/blob/master/docs/compass/04-01-installation.md) or to the [Director GraphQL schema](https://github.com/kyma-incubator/compass/blob/master/components/director/pkg/graphql/schema.graphql) directly. 
```

Comment:
Maybe we should add a link to Compass Connector as well.

Reply:
I think it might be confusing as we already use `Connector` a lot in Application Connectivity, plus the CAUTION note already links to the Compass repo and from there, anyone interested can navigate to its subcomponents.